ddtrace/span.py

```diff
@@ -482,11 +484,9 @@ def pprint(self):
             ("end", "" if not self.duration else self.start + self.duration),
             ("duration", "%fs" % (self.duration or 0)),
             ("error", self.error),
-            ("tags", ""),
+            ("tags", ",".join("%r:%s" % kv for kv in sorted(self.meta.items()))),
```

Comment:
can we add `metrics` as well here?

Reply:
anything wrong with keeping it as a `dict`? I don't think that will cause much issues?

Reply:
> can we add `metrics` as well here?

sure, was going to leave it for https://github.com/DataDog/dd-trace-py/pull/2244 but can add it here if @P403n1x87 is cool with it.

Reply:
> anything wrong with keeping it as a `dict`? I don't think that will cause much issues?

wanted to make sure we get the same ordering across Python versions.

Reply:
@majorgreys sure I'm happy for #2244 to be included in here 👍

Reply:
added!

Reply:
> > anything wrong with keeping it as a `dict`? I don't think that will cause much issues?
> 
> wanted to make sure we get the same ordering across Python versions.

Ordering of the tags by key name you mean?

Could use an `OrderedDict` here too.

I don't have the biggest concern one way or the other, do we think there could ever be formatting issues? (`"%r:%s" % kv`)

Reply:
Sorry I was confusing this dict question with when we need to pass it to `json.dumps`. Here we want a consistent ordering and given we don't the ordering of tags being added we sort to keep things consistent.

> do we think there could ever be formatting issues?

Typing of `meta` requires we use `"%r:%s"` given the `_MetaKeyType` we have defined.